src/amo/components/SearchResult/styles.scss

```diff
@@ -18,6 +18,7 @@ $icon-default-size: 32px;
   display: flex;
   flex-flow: row wrap;
   margin: 0;
+  max-width: $theme-width-default;
```

Comment:
this will affect extension but I thought it was better to be consistent (for when there is a mix)